pkg/controllers/controller.go

```diff
@@ -49,22 +49,24 @@ func (c *GenericController) Reconcile(ctx context.Context, req reconcile.Request
 	}
 	// 2. Copy object for merge patch base
 	persisted := resource.DeepCopyObject()
-	// 3. Set to true to remove race condition
+	// 3. Set Defaults
```

Comment:
Would be nice to explain why here - I could see eventually someone removes this code because they think it's redundant

Reply:
Good call. Will do